cerbottana/plugins/__init__.py

```diff
@@ -1,3 +1,5 @@
+# Author: Plato (palt0)
```

Comment:
Non so se tu voglia specificare qualcosa in più, perché il refactor dei decoratori (quindi questo modulo e in generale la logica di base) è stato scritto da me, ma in seguito è stato aggiunto il codice per le htmlpages e qualche parametro in più. Credo vada bene così ma mi sembrava giusto lasciare un commento per sicurezza.